src/test/java/com/iexec/sms/authorization/AuthorizationServiceTests.java

```diff
@@ -51,92 +51,92 @@ class AuthorizationServiceTests {
     // region isAuthorizedOnExecutionWithDetailedIssue
     @Test
     void shouldBeAuthorizedOnExecutionOfTeeTaskWithDetails() {
-        ChainDeal chainDeal = getChainDeal();
-        ChainTask chainTask = TestUtils.getChainTask(ACTIVE);
+        final ChainDeal chainDeal = getChainDeal();
+        final ChainTask chainTask = TestUtils.getChainTask(ACTIVE);
         WorkerpoolAuthorization auth = TestUtils.getTeeWorkerpoolAuth();
```

Comment:
Some `final WorkerpoolAuthorization` are missing in several test methods
```suggestion
        final WorkerpoolAuthorization auth = TestUtils.getTeeWorkerpoolAuth();
```